basyx.components/basyx.components.docker/basyx.components.registry/src/main/java/org/eclipse/basyx/components/registry/RegistryComponent.java

```diff
@@ -195,7 +198,7 @@ private BaSyxMongoDBConfiguration loadMongoDBConfiguration() {
 	}
 
 	private HttpServlet createRegistryServlet() {
-		if (this.registryConfig.isTaggedDirectoryEnabled()) {
+		if (!this.registryConfig.isTaggedDirectoryEnabled()) {
```

Comment:
This change is wrong, it enables the taggedDirectory feature whenever isTaggedDirectoryEnabled() returns true